tests/nemo_text_processing/sv/data_text_normalization/test_cases_normalize_with_audio.txt

```diff
@@ -0,0 +1,8 @@
+~1
+en
+ett
+~21
+tjugoen
+tjugoett
+tjugo en
+tjugo ett
```

Comment:
how about cases like "21" -> "two one"? these all seem to have "twenty" or is this not used in Swedish?
There are many classes with "non deterministic" conditions, could you expand the test cases here to cover those?

Reply:
I guess it could come up; I'll add it.
I meant to expand the non deterministic tests, and forgot about it. I'll take a look at it tomorrow.

Reply:
I can't get the tests to pass, so I've removed audio-based normalisation for now

FWIW, I've tried increasing `n_tagged`, I've tried setting it to -1, and I've tried adding a low weight on all of the non-deterministic parts of the cardinal tagger, and nothing seems to make much of a difference (though -1 fails entirely). If you have any other ideas of things I could try, I'd love to hear them.

Reply:
There was a bug in non-deterministic setup for non-Eng; this is was I'm getting for `--text=21` now:
```
$ python normalize_with_audio.py --cache_dir=cache_dir/ --text="21" --language=sv
Created cache_dir/sv_tn_False_deterministic_cased__tokenize.far
Created cache_dir/sv_tn_False_deterministic_verbalizer.far
tjugi en
tjugiett
tjugo en
tjugo ett
tjugi ett
tjugoen
tjugien
tjugoett
två ett
21
```